SQL/paradise_schema.sql

```diff
@@ -277,7 +277,7 @@ CREATE TABLE `player` (
   `volume_mixer` longtext COLLATE utf8mb4_unicode_ci DEFAULT NULL,
   `lastchangelog` varchar(32) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '0',
   `exp` longtext COLLATE utf8mb4_unicode_ci DEFAULT NULL,
-  `clientfps` smallint(4) DEFAULT '0',
+  `clientfps` smallint(4) DEFAULT '63',
```

Comment:
You must follow standard SQL update procedures, including:
- Bumping schema number in `code/__DEFINES/misc.dm#370`
- Bumping schema number in the default config at `config/example/config.toml#148`
- Creating an update file in `SQL/updates/34-35.sql` with this change that can be applied to the production schema